test/functional/terminal/helpers.lua

```diff
@@ -30,10 +30,13 @@ local function clear_attrs() feed_termcode('[0;10m') end
 -- mouse
 local function enable_mouse() feed_termcode('[?1002h') end
 local function disable_mouse() feed_termcode('[?1002l') end
+local function print_screen_size()
+  helpers.sleep(1000)
```

Comment:
1s sleep in the tests is pretty brutal, and this is used in many tests... can you use `helpers.retry()` instead? See `test/functional/autocmd/termclose_spec.lua` for example.

Note: retry() depends on an error being thrown--it won't retry on test "failure". (Again see `test/functional/autocmd/termclose_spec.lua`)